lib/flood_risk_engine/engine.rb

```diff
@@ -6,6 +6,9 @@ module FloodRiskEngine
   class Engine < ::Rails::Engine
     isolate_namespace FloodRiskEngine
 
+    # Add a load path for this specific Engine
+    config.autoload_paths += Dir["#{config.root}/lib/**/"]
```

Comment:
I'm not sure having some code in `lib` and some in `app` is a good idea. Why not just keep everything together in `app`